activerecord/test/cases/primary_keys_test.rb

```diff
@@ -351,9 +351,9 @@ class Widget < ActiveRecord::Base
     test "schema dump primary key with bigserial" do
       schema = dump_table_schema "widgets"
       if current_adapter?(:PostgreSQLAdapter)
-        assert_match %r{create_table "widgets", id: :bigserial}, schema
+        assert_match %r{create_table "widgets", id: :bigserial, force: :cascade}, schema
       else
-        assert_match %r{create_table "widgets", id: :bigint}, schema
+        assert_match %r{create_table "widgets", id: :bigint, force: :cascade}, schema
```

Comment:
If we want to make sure `limit: 8` is not there I think so, otherwise it can be after the `force: :cascade` option and this test will not catch.